autoreject/autoreject.py

```diff
@@ -397,17 +403,22 @@ class LocalAutoReject(BaseAutoReject):
         The sensor-level thresholds with channel names as keys
         and the peak-to-peak thresholds as the values.
     """
-    def __init__(self, thresh_func=None, consensus_perc=0.1,
-                 n_interpolate=0, method='bayesian_optimization',
+
+    def __init__(self, consensus_perc=0.1,
+                 n_interpolate=0, thresh_func=None,
+                 method='bayesian_optimization',
                  picks=None,
                  verbose='progressbar'):
+        """Init it."""
         if thresh_func is None:
             thresh_func = compute_thresholds
         if not (0 <= consensus_perc <= 1):
             raise ValueError('"consensus_perc" must be between 0 and 1. '
                              'You gave me %s.' % consensus_perc)
-        self.consensus_perc = consensus_perc
-        self.n_interpolate = n_interpolate
+        self.consensus_perc = {
+            ch: consensus_perc for ch in ('mag', 'grad', 'eeg')}
+        self.n_interpolate = {
+            ch: n_interpolate for ch in ('mag', 'grad', 'eeg')}
```

Comment:
let's handle that later. I just wanted it to be a dict by default and avoid updating / setting stuff during fit.